src/gui/mods/list/ModListLayer.cpp

```diff
@@ -251,8 +256,9 @@ void ModListLayer::onSearchFilters(CCObject*) {
 	SearchFilterPopup::create(this)->show();
 }
 
-ModListLayer* ModListLayer::create() {
+ModListLayer* ModListLayer::create(CCLayer* parentLayer) {
 	auto ret = new ModListLayer();
+    ret->parentLayer = parentLayer;
```

Comment:
`parentLayer` should be passed as a parameter to `ModListLayer::init` and assigned to the member there